src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -624,6 +624,22 @@ private FormulaValue GetVariableOrFail(ResolvedObjectNode node, ISymbolSlot slot
                 var value = _symbolValues.Get(slot);
                 if (value != null)
                 {
+                    DateTime dateTime = default;

```

Comment:
>DateTime dateTime = default; [](http://example.com/codeflow?start=20&length=28)

This is the right idea, but we need to do this check in a more common place. The problem is that there are other paths to get a DateValue beside just symbol lookup (say field access). 

Maybe a helper directly on DateValue,DateTimeValue like 
"GetTimeInUTC(Engine engine)"

- Note it will need to take some extra param (like Engine) to get the TZI. 
- The benefit of putting it directly on Date*Value, especially next to existing Value, is that it's easier to see it and make sure it's getting called everywhere.

Reply:
Added a new Helper.